diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/LineLength.kt

```diff
@@ -282,7 +283,22 @@ class LineLength(configRules: List<RulesConfig>) : DiktatRule(
         val incorrectText = wrongStringTemplate.node.text
         val firstPart = incorrectText.substring(0, wrongStringTemplate.delimiterIndex)
         val secondPart = incorrectText.substring(wrongStringTemplate.delimiterIndex, incorrectText.length)
-        wrongStringTemplate.node.treeParent.replaceChild(wrongStringTemplate.node, KotlinParser().createNode("$firstPart\" +\n\"$secondPart"))
+        val correctNode =
+            try {
+                KotlinParser().createNode("$firstPart\" +\n\"$secondPart")
```

Comment:
I think we need to just not put any pluses in ```${}``` block, no?
Need to check if we are in `${}` write now or not. If yes - add a new line

Reply:
I wanted to suggest the same. We can find the current node by offset, and if it is text - add concatenation, else - simply add a line break